hw/opentitan/ot_clkmgr.c

```diff
@@ -1129,7 +1130,13 @@ static void ot_clkmgr_write(void *opaque, hwaddr addr, uint64_t val64,
         s->regs[reg] &= val32;
         break;
     case R_JITTER_ENABLE:
-        if (s->regs[R_JITTER_REGWEN]) {
+        /**
+         * A known bug in the Earlgrey 1.0.0 clkmgr is that the Jitter Enable
+         * REGWEN was not connected properly, meaning that it is ignored here
+         * in the Earlgrey 1.0.0 version of the device.
+         */
+        if (s->regs[R_JITTER_REGWEN] ||
+            s->version == OT_CLKMGR_VERSION_EG_1_0_0) {
```

Comment:
I wonder if we should add a guest warning for these kinds of hardware bugs. QEMU is intended to help SW developers after all.